app/models/content_item.rb

```diff
@@ -45,7 +45,7 @@ class ContentItem < ActiveRecord::Base
   validates :details, well_formed_content_types: { must_include: "text/html" }
 
   def requires_base_path?
-    EMPTY_BASE_PATH_FORMATS.exclude?(document_type)
+    EMPTY_BASE_PATH_FORMATS.exclude?(document_type || format)
```

Comment:
In my head, I was not sure what the interaction with this model, and the data coming in to the system is. I've checked now, and the document_type is coped with in PutContent, so I've removed the commit adding this.